client/src/pages/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -1,82 +1,91 @@
     const newFile = event.target.files![0];
     setFile(newFile);
   };
 
   return (
-    <Grid className={classes.root}>
-      <Box>
-        <Typography className={classes.pageTitle} variant="h1">
-          Profile Photo
-        </Typography>
-      </Box>
-      <Box className={classes.root}>
-        <Avatar className={classes.avatarImage} alt="Profile Image" src={avatar} />
-        <Typography className={classes.subtext} variant="subtitle1">
-          Be sure to use a photo that clearly shows your face
-        </Typography>
-      </Box>
-      <Box>
-        <form action="/image/avatar" encType="multipart/form-data" method="POST">
-          <input
-            accept="image/*"
-            id="uploadImageButton"
-            type="file"
-            name="avatar"
-            multiple={false}
-            className={classes.input}
-            onChange={handleUpload}
-          />
-          <label htmlFor="uploadImageButton">
-            <Button
-              className={classes.uploadButton}
-              variant="outlined"
-              color="primary"
-              component="span"
-              onSubmit={handleUpload}
-            >
-              Upload a file from your device
-            </Button>
-          </label>
-        </form>
-      </Box>
-      <Box>
-        <Grid item className={classes.delete}>
-          <Button startIcon={<DeleteIcon />}>Delete Photo</Button>
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
+      <Grid container>
+        <Grid item>
+          <MyProfileSideBanner profile={profile as Profile} user={user as User} />
+        </Grid>
+        <Grid item className={classes.card}>
+          <Card>
+            <Typography className={classes.pageTitle} align="center" variant="h1">
+              Profile Photo
+            </Typography>
+            <Avatar className={classes.avatarImage} alt="Profile Image" src={avatar} />
+            <Typography className={classes.subtext} align="center" variant="subtitle1">
+              Be sure to use a photo that clearly shows your face
+            </Typography>
+            <form action="/image/avatar" encType="multipart/form-data" method="POST" className={classes.form}>
+              <input
+                accept="image/*"
+                id="uploadImageButton"
+                type="file"
+                name="avatar"
+                multiple={false}
+                className={classes.input}
+                onChange={handleUpload}
+              />
+              <label htmlFor="uploadImageButton">
+                <Button
+                  className={classes.uploadButton}
+                  variant="outlined"
+                  color="primary"
+                  component="span"
+                  onSubmit={handleUpload}
+                >
+                  Upload a file from your device
+                </Button>
+              </label>
+            </form>
```

Comment:
Is it possible to utilize [Formik](https://formik.org/docs/overview)? It is already included in the project.

Reply:
I'm not familiar with Formik, but I will look into it today.